packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -116,6 +116,31 @@ const propTypes = {
    * The selected or entered date value to display in the date input.
    */
   value: PropTypes.string,
+  /**
+   * @private
+   * An array of ISO 8601 string representation of the dates to disable in the picker. The values must be in the `YYYY-MM-DD` format.
+   */
+  excludeDates: PropTypes.arrayOf(PropTypes.string),
+  /**
+   * @private
+   * An array of ISO 8601 string representation of the dates to enable in the picker. All Other dates will be disabled. The values must be in the `YYYY-MM-DD` format.
+   */
+  includeDates: PropTypes.arrayOf(PropTypes.string),
+  /**
+   * @private
+   * An ISO 8601 string representation of the maximum date that can be selected. The value must be in the `YYYY-MM-DD` format. Must be on or before `12/31/2100`.
+   */
+  maxDate: PropTypes.string,
+  /**
+   * @private
+   * An ISO 8601 string representation of the minimum date that can be selected. The value must be in the `YYYY-MM-DD` format. Must be on or after `01/01/1900`
+   */
+  minDate: PropTypes.string,
+  /**
+   * A function that gets called for each date in the picker to evaluate which date should be disabled.
+   * A return value of true will be enabled and false will be disabled.
+   */
+  filterDate: PropTypes.func,
```

Comment:
We should mention this prop addition in the CHANGELOG.

Reply:
Would it be required as we won't make it available for consumers, only being used as a passed down prop as mentioned above